v1/ocagent_test.go

```diff
@@ -0,0 +1,313 @@
+// Copyright 2018, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ocagent_test
+
+import (
+	"context"
+	"fmt"
+	"net"
+	"os"
+	"strconv"
+	"strings"
+	"testing"
+	"time"
+
+	"contrib.go.opencensus.io/exporter/ocagent/v1"
+	commonpb "github.com/census-instrumentation/opencensus-proto/gen-go/agent/common/v1"
+	agenttracepb "github.com/census-instrumentation/opencensus-proto/gen-go/agent/trace/v1"
+	tracepb "github.com/census-instrumentation/opencensus-proto/gen-go/trace/v1"
+	"go.opencensus.io"
+	"go.opencensus.io/trace"
+)
+
+func TestNewExporter_endToEnd(t *testing.T) {
+	ma := runMockAgent(t)
+	defer ma.stop()
```

Comment:
nit: is this needed, given that `ma` is already stopped at https://github.com/census-ecosystem/opencensus-go-exporter-ocagent/pull/7/files#diff-97b1421ac9453b85af7c3b7cf4907786R143?

Reply:
It's just there in case we return early and forget to manually run `ma.stop()`